grand_challenge_forge/partials/example-evaluation-method/evaluate.py.j2

```diff
@@ -1,19 +1,29 @@
 """
 The following is a simple example evaluation method.
 
-It is meant to run within a container.
+It is meant to run within a container. Its steps are as follows:
+
+  1. Read the algorithm output
+  2. Associate original algorithm input with a ground truth via predictions.json
+  3. Calculate metrics by comparing the algorithm output to the ground truth
+  4. Repeat steps 1, 2, and 3 for all algorithm outputs
+  5. Aggregate the calculated metrics
+  6. Save the metrics to metrics.json
 
 To run it locally, you can call the following bash script:
 
   ./do_test_run.sh
 
-This will start the evaluation, reads from ./test/input and outputs to ./test/output
+This will start the inference and reads from ./test/input and writes to ./test/output
```

Comment:
I liked the original term "evaluation" better than "inference", just my 2 cents.